core/shared/platform/windows/win_util.c

```diff
@@ -69,17 +70,17 @@ uwp_print_to_debugger(const char *format, va_list ap)
     char *buf = stack_buf;
     int ret = vsnprintf(stack_buf, sizeof(stack_buf), format, ap);
 
-    if (ret >= sizeof(stack_buf)) {
+    if ((size_t)ret >= sizeof(stack_buf)) {
         // Allocate an extra byte for the null terminator.
-        char *heap_buf = BH_MALLOC(ret + 1);
+        char *heap_buf = BH_MALLOC((unsigned int)(ret) + 1);
         buf = heap_buf;
 
         if (heap_buf == NULL) {
             errno = ENOMEM;
```

Comment:
Should we use OutputDebugStringA to print the stack_buf (which is part of the whole buffer)?

Reply:
good idea, will change it to output stack_buf